ios/VibeTunnel/Shared.xcconfig

```diff
@@ -4,15 +4,14 @@
 // Include version configuration
 #include "version.xcconfig"
 
-// Include local development settings (if exists)
+// Default values (used when Local.xcconfig doesn't exist)
+DEVELOPMENT_TEAM = Y5PE65HELJ
```

Comment:
Please read the README and use the local.xcconfig setup.

Reply:
Yes, sorry, CC added this change but it's not needed. I just needed to remove hardcoded values from pbxproj so it uses local properly.